### PR TITLE
Fix typos and disorienting example in formater readme.md

### PR DESCRIPTION
**Pull request by illi-kun**
@mugling please review it (was `CONTANER` misspelled unintentionally or it's something I don't get?).

### CODE REVIEW

File: tools/format/README.md [hunk 1]
```diff
@@ -15,11 +15,11 @@ external dependencies beyond the p5-JSON module
 
 Parse a file and output JSON in canonical format
 
-`format.pl examples/vehicle.json > output.json`
+`format.pl examples/vehicle_part.json > output.json`
```
Comment (mugling) [inline tools/format/README.md]: Good idea to make them refer to actual files

**Comment by mugling** [comment]
> was CONTANER misspelled unintentionally or it's something I don't get?

It was as subtle ploy to see if anyone reads documentation :smile:

**Comment by mugling** [comment]
Was the documentation useful?

**Comment by illi-kun** [comment]
Yes, with it I was able to run the check & get some results.

The info I'm not able to find is how to use lint-make properly, and also, I wonder why you're naming required module as 'p5-JSON' (I unsuccessfully tried to find this name in package managers in Linux Mint installed on my backup laptop but narc's comment about ```libjson-perl``` helps me to solve this puzzle).

Anyway, using of these new tools is useful but difficult, so more docs are welcomed.

**Comment by mugling** [comment]
The module naming is a BSD idiom. Happy to go with the linuxism in the docs as it's certainly the more common OS.

There is some initial documentation for the makefile targets in `json_whitelist`. Next on my list is a GH wiki page, extending coverage to include flags and then working on a fully cross-platform solution

**Comment by illi-kun** [comment]
> GH wiki page

Mixup of traditional readme.md files and GH wiki (which is supported only by you, afaik) is another source of confuse ;-)

Anyway, hints anywhere is better than no hints at all, so go on!

**Comment by mugling** [comment]
I did bully @Coolthulhu into writing a page once...

**Pull request merged**